client/my-sites/plans-features-main/index.jsx

```diff
@@ -619,6 +628,11 @@ export default connect(
 			currentPlan,
 		} );
 
+		const isReskinned =
+			props.isInSignup &&
+			'onboarding' === props.flowName &&
+			'reskinned' === abtest( 'reskinSignupFlow' );
```

Comment:
Can you replace `abtest()` with `getABTestVariation()`? Since the signup main component already handles the `abtest()` assignment, you don't need to check it here. The following code would be enough.

```suggestion
		const isReskinned = ( 'reskinned' === getABTestVariation('reskinSignupFlow') ); 
```

Reply:
I already made a note about the way of detection test group. Please check the `Notes` section.
https://github.com/Automattic/martech/issues/578

Reply:
I already made a note about the way of detection test group. Please check the `Notes` section in the master issue.
578-gh-martech

Reply:
Thanks! Since the `PlanFeaturesMain` component is used in other places as well, I have added an extra condition to check if we are in the signup flow: `props.isInSignup && 'onboarding' === props.flowName`. Is this the correct way to check this?

Reply:
@aneeshd16 Though it's correct and recommended in general, it's not necessary in this a/b test. Since the main signup layout already check the flow name, users are never assigned to the `reskinned` group uless they are not in the `onboarding` signup flow.